src/App.php

```diff
@@ -352,7 +352,7 @@ public function terminate($output = '', array $headers = []): void
             $headers['content-type'] = $this->response_headers['content-type'];
         }
 
-        $type = preg_replace('~;.*~', '', strtolower($headers['content-type'])); // in LC without charset
+        $type = preg_replace('~;.*~', '', mb_strtolower($headers['content-type'])); // in LC without charset
```

Comment:
content type (and headers in general) are strictly taken ascii only, I would never use mb_ here